src/app/oktaAuthCallback/oktaAuthCallback.component.js

```diff
@@ -37,7 +37,7 @@ class OktaAuthCallbackController {
   }
 
   initializeVariables () {
-    this.noticeToUser = 'Authenticating...'
+    this.isReadyToRedirect = false
     this.isLoading = true
```

Comment:
I don't see `this.isLoading` being read anywhere. Am I missing something?

Reply:
It is being used on the HTML

Reply:
I don't see `isLoading` in `src/app/oktaAuthCallback/oktaAuthCallback.tpl.html`, just `isReadyToRedirect`.

Reply:
Sorry, you're correct. I was looking at the Sign Out component.